README.md

```diff
@@ -1,9 +1,17 @@
 # quetz-client
 
-A Python client to interact with a Quetz server.
+A Python client to interact with a Quetz server. Compatible with `quetz>=0.4.0`.
```

Comment:
Since we're switching to the new upload endpoint, we're not compatible with 0.4.0 anymore, right?

Reply:
Yes, paste mistake 👌🏼